packages/react-ai/src/components/AIConversation/views/ConversationView.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useElement } from '../context/elements';
+
+export default function Conversation(): JSX.Element {
+  const View = useElement('View');
+  const Paragraph = useElement('Text');
+  return (
```

Comment:
Just for my own understanding, is `Conversation` the root component?

Reply:
Yep that was the idea